Lychee

mre opened the pull request:
Since the raw input files contain Golang templates, we check the output rendered into public instead.
The two pages sms-digital.com and LinkedIn seem to return false-positive errors and were excluded for the time being.

andygrunwald commented:
Looooove :heart: + :beers:

andygrunwald commented:
... aaaaand deployed. See http://andygrunwald.com/.
Thaaank you @creinartz!

And regarding your SCSS comment: I will switch to SCSS. But at first i want to go online and stripped features out. You know? Make it work. Make it fast. Make it beautiful :)

mre committed:
Add lychee

Since the raw input files contain Golang templates, we check the output rendered into public instead.
The two pages sms-digital.com and LinkedIn seem to return false-positive errors and were excluded for the time being.

mre committed:
Update links.yml

andygrunwald reviewed (approved):
Whoop whoop!
